src/Offer/ReadModel/JSONLD/PropertyPolyfillOfferRepository.php

```diff
@@ -235,8 +238,9 @@ function (array $json) {
                     return $json;
                 };
 
-                $json = $filterNullLabels($json, 'labels');
-                $json = $filterNullLabels($json, 'hiddenLabels');
+                foreach (self::FILTERED_NULL_LABELS as $filteredNullLabel) {
+                    $json = $filterNullLabels($json, $filteredNullLabel);
```

Comment:
Also the method name can be more general, maybe something like `removeEmptyArray`

Reply:
While having a closer look, this can be dangerous to use in the context of bookingInfo because the `array_filter`

Reply:
I have made it into a separate method